sdk/servicebus/azure-messaging-servicebus-track2-perf/src/main/java/com/azure/messaging/servicebus/perf/ServiceBusStressOptions.java

```diff
@@ -22,6 +22,15 @@ public class ServiceBusStressOptions extends PerfStressOptions {
     @Parameter(names = { "-msb", "--messageSizeBytes" }, description = "Size(in bytes) of one Message")
     private int messagesSizeBytesToSend = 10;
 
+    @Parameter(names = { "-idm", "--isDeleteMode" }, description = "Receiver client is receive_and_delete mode or peek_lock mode")
+    private boolean isDeleteMode = true;
+
+    @Parameter(names = { "-mcc", "--maxConcurrentCalls" }, description = "Processor client max concurrent calls")
+    private int maxConcurrentCalls = 1;
+
+    @Parameter(names = { "-mst", "--messageSendTimes" }, description = "Messages send times")
```

Comment:
Personally I won't know what "message send times" means. Be more specific in description?

And what is the difference to "messagesToSend"?

Reply:
use `messageBatchSize` and `messageBatchSendTimes` to send billion messages.